miasm2/arch/x86/sem.py

```diff
@@ -2416,9 +2459,65 @@ def rdtsc(ir, instr):
     return e, []
 
 
-# XXX TODO
 def daa(ir, instr):
-    return [], None
+    e = []
```

Comment:
Here, you take the time to use variables with expressive names, but you keep `e` for the output.
You could, for instance, replace it with `affblock`.